lua/tardis/screens/sh_fallbackscreen.lua

```diff
@@ -34,32 +58,48 @@ TARDIS:AddScreen("New features fallback", {menu=false}, function(self,ext,int,fr
     power:SetFont("TARDIS-Default")
     power.DoClick = function()
         TARDIS:Control("power")
+    end    
+    
+    local repair=vgui.Create("DButton",frame)
+    repair:SetSize( frame:GetWide()*0.2, frame:GetTall()*0.2 )
+    repair:SetPos(frame:GetWide()*0.35 - repair:GetWide()*0.5,frame:GetTall()*0.15 - repair:GetTall()*0.5)
+    repair:SetText("Repair TARDIS")
+    repair:SetFont("TARDIS-Default")
+    repair.DoClick = function()
+        TARDIS:Control("repair")
     end
 
     local fastremat=vgui.Create("DButton",frame)
     fastremat:SetSize( frame:GetWide()*0.2, frame:GetTall()*0.2 )
     fastremat:SetPos(frame:GetWide()*0.13 - fastremat:GetWide()*0.5,frame:GetTall()*0.4 - fastremat:GetTall()*0.5)
-    fastremat:SetText("Toggle Fast Remat")
+    fastremat:SetText("Fast Remat PLH")
     fastremat:SetFont("TARDIS-Default")
     fastremat.DoClick = function()
         TARDIS:Control("fastremat")
     end
+    function fastremat:Think()
+        local on = ext:GetData("demat-fast")
+        if on then
+            self:SetText("Fast Remat on")
+        else
+            self:SetText("Fast Remat off")
+        end
+    end
     
-    local fastrematlbl = vgui.Create("DLabel",frame)
-    fastrematlbl:SetTextColor(Color(0,0,0))
-    fastrematlbl:SetFont("TARDIS-Med")
-    fastrematlbl.DoLayout = function(self)
-        fastrematlbl:SizeToContents()
-        fastrematlbl:SetPos((frame:GetWide()*0.55)-(fastrematlbl:GetWide()*0.5),(frame:GetTall()*0.4)-(fastrematlbl:GetTall()*0.5))
+    local physbrake = vgui.Create("DButton",frame)
+    physbrake:SetSize( frame:GetWide()*0.2, frame:GetTall()*0.2)
+    physbrake:SetFont("TARDIS-Default")
+    physbrake:SetPos((frame:GetWide()*0.35)-(physbrake:GetWide()*0.5),(frame:GetTall()*0.4)-(physbrake:GetTall()*0.5))
+    physbrake:SetText("Physlock PLH")
```

Comment:
PLH again, not sure on the meaning

Reply:
Made up short for "placeholder" because I didn't want to write "placeholder"